### PR TITLE
add captcha bundle to display bundle

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
         "open-orchestra/open-orchestra-front-bundle": "dev-master",
         "open-orchestra/open-orchestra-user-bundle": "dev-master",
         "open-orchestra/open-orchestra-media-bundle": "dev-master",
-        "open-orchestra/open-orchestra-model-bundle": "dev-master"
+        "open-orchestra/open-orchestra-model-bundle": "dev-master",
+        "innocead/captcha-bundle": "*"
```
Comment (itkg-rboessel) [inline composer.json]: not work when i put it in display bundle composer.json

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/147

File: app/config/config.yml [hunk 2]
```diff
@@ -126,3 +126,13 @@ fos_http_cache:
...
+
+innocead_captcha:
+    width: 150
+    height: 30
+    max_chars: 6
+    min_chars: 5
+    bg_transparent: false                     # transparent background
+    bg_red: 255                              # used if background is not transparen
+    bg_green: 255
+    bg_blue: 255
```
Comment (itkg-ngilain) [inline app/config/config.yml]: newline at EOF

File: app/config/config.yml [hunk 3]
```diff
@@ -126,3 +126,13 @@ fos_http_cache:
...
             base_url: "%fos_http_cache.proxy_client.varnish.base_url%"
         default: varnish
+
+innocead_captcha:
+    width: 150
+    height: 30
+    max_chars: 6
+    min_chars: 5
+    bg_transparent: false                     # transparent background
+    bg_red: 255                              # used if background is not transparen
```
Comment (itkg-ngilain) [inline app/config/config.yml]: transparent more than transparen ^^

File: composer.json (hunk 1 above)
Comment (itkg-ngilain) [inline composer.json]: What is the problem when put in the displaybundle : no installation with composer, or an error message when displaying the site, or no error but no captcha ?

Comment (itkg-rboessel) [inline composer.json]: no installation with composer

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**